octobot_trading/exchanges/connectors/exchange_simulator.py

```diff
@@ -208,7 +209,7 @@ def get_time_frames(self, importer):
                                                         set(self.exchange_manager.exchange_config.traded_time_frames)),
                                                    reverse=True)
 
-    def get_split_pair_from_exchange(self, pair) -> (str, str):
+    def get_split_pair_from_exchange(self, pair) -> typing.Tuple[str, str]:
```

Comment:
Why not keep as it?

Reply:
black in VSCode lints it as an error and suggests this change, but with black in pycharm it doesnt care

Reply:
Ok but it's not a good practice to update a code  that you have not modified. Can you revert it please?

Reply:
I changed it back